_sass/minimal-mistakes/_base.scss

```
@@ -128,6 +131,63 @@ blockquote {
   }
 }
 
+/* Block quote with bigger quotes*/
+blockquote.highlight-quote {
+  border: none;
+  font-family: Georgia, "Times New Roman", Times, serif;
+  margin-bottom: 90px;
+  margin-top: 110px;
+  padding: 0 80px 0 30px;
+  max-width: 35rem;
+  //quotes: "\201C""\201D""\2018""\2019";
+
+  footer {
+    text-align: left;
+    font-style: italic;
+    font-weight: 500;
+  }
+
+  p {
+    font-size: 21px;
+    position: relative;
+    margin-bottom: 10px;
+
+    &:before,
+    &:after {
+      position: absolute;
+      font-weight: bold;
+      font-size: 100px;
+      color: $pyos-teal;
+      line-height: 0;
+    }
+
+    &:before {
+      content: open-quote;
+      left: 0;
+      top: -20px;
+    }
+
+    &:after {
+      content: close-quote;
+      right: 0;
+      bottom: -20px;
+    }
+  }
+  // Success class (magenta)
+  &.magenta p {
+    &:before,
+    &:after {
+      color: $pyos-magenta;
+    }
+  }
+  &.purple p {
+    &:before,
+    &:after {
+      color: pyos-mediumpurple;
```

```suggestion
      color: $pyos-mediumpurple;
```

this shouldn't work - why didn't it throw an error?

---

for the record - it's syntactically valid CSS because you can specify color as a keyword like, it's just that there is no keyword color pyos-mediumpurple. so probably a linter would complain but sass would compile it

---

Oh wow - that is interesting. sass has SO MANY features that I didn't know about until this past year. I really like it.

---

it's definitely the way to go imo. vanilla CSS has adopted a lot of sass things, including [nesting](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_nesting/Using_CSS_nesting) which is one of the major reasons i like it,l but partials and extend rocks